Changelog

```diff
@@ -1,8 +1,69 @@
 Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v4.2.0
+    - FEATURE: Enforce C++14 requirement (Chocobo1)
     - FEATURE: Add OpenSSL version to GUI and stackdump (Chocobo1)
     - FEATURE: Use PBKDF2 for the GUI lock. You will need to set again your password. (Chocobo1)
+    - FEATURE: Rename "#" column to "Tier" in the tracker list (thalieht)
+    - FEATURE: Lots of code refactorings, cleanups, improvements and optimizations (Chocobo1, glassez, thalieht)
+    - FEATURE: Allow setting larger checking memory usage in GUI (airium)
+    - FEATURE: Converted remaining icons to svg (Bert Verhelst)
+    - BUGFIX: Avoid performance penalty when logger is full (Chocobo1)
+    - BUGFIX: Remove the max half-open connections option from GUI (thalieht)
```

Comment:
I think this needs to be more informative i.e. "Remove the **leftover** max half-open..." because otherwise it may imply the option is still present in the config file.

Reply:
Most of these are verbatim the commit message. I will change it to
```
Remove the max half-open connections option
```